sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/Schema.java

```diff
@@ -222,6 +224,9 @@ public boolean isDateType() {
     public boolean isContainerType() {
```

Comment:
Is map a container type as well? A composite type?

Reply:
while, my thought is MAP-> MAP, ARRAY -> CONTAINER, ROW -> COMPOSITE, to make it clear for the backend types.